Update documentation files (#313)
vector-im/element-x-ios: Tools/Scripts/README.md

```
./bootTestSimulator.py 'iPhone 13 Pro Max'
```
## Create screen templates

> New screen flows are currently using MVVM-Coordinator pattern. Run [Tools/Scripts/createScreen.sh](Tools/Scripts/createScreen.sh) to create a new screen or a new screen flow.

Reviewer: ```suggestion
New screen flows are currently using the the MVVM-Coordinator pattern. Run [Tools/Scripts/createScreen.sh](Tools/Scripts/createScreen.sh) to create a new screen and all its required dependencies.
```
